src/theme/components/StatusBage.tsx

```diff
@@ -0,0 +1,25 @@
+import { tagAnatomy } from '@chakra-ui/anatomy'
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+
+const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(tagAnatomy.keys)
+
+const vocdoniStatusBadge = definePartsStyle({
+  container: {
+    padding: '4px 12px',
+    borderRadius: '4px',
+    height: '18px',
+    fontStyle: 'normal',
+    fontWeight: '700',
+    fontFamily: 'Manrope',
+    fontSize: '12px',
+    lineHeight: '150%',
+    color: '#fff',
+    bg: 'rgb(35, 114, 116)',
+  },
+})
+
+export const tagTheme = defineMultiStyleConfig({
```

Comment:
It is a bit confusing to have the Tag theme under the StatusBadge theme file (btw there's a typo on the filename, I've just noticed.. it's missing a `d`).

I'd name the file `Tag.tsx` and also the `tagTheme` would be `Tag` (this way matches with the file name and also can be imported easily, since it's the name used by the component in the theme).

Reply:
Absolutely agree. As before, this code is copy pasted from the old explorer. I fix it: 0adccc2